packaging/installer/kickstart.sh

```diff
@@ -1742,6 +1743,12 @@ while [ -n "${1}" ]; do
       NETDATA_INSTALLER_OPTIONS="${NETDATA_INSTALLER_OPTIONS} --disable-telemetry"
       ;;
     "--install")
+      warning "--install flag is deprecated and will be removed in the future version."
+      warning "Please use --install-prefix instead."
```

Comment:
Due to how deferred warnings are printed at the end of a run, having two lines here will result in two separate bullet points for the version. Long term we ideally need to fix that, but short term it’s preferable to just put the full warning text on one line for this type of thing.